### ISSUE TITLE
After updating to 1.01 FlyPhotos doesn't open anymore

### ISSUE THREAD

**Issue by Skyyblaze** [riyasy/FlyPhotos#5]
No matter if I try to launch it directly or try to open an image, FlyPhotos doesn't open anymore. It does not show up in Task Manager either and there's no error message. I'm using the version without .NET Runtime included and version 1.0 works fine. A reinstall didn't help sadly. My Windows version is Windows 11 Pro 23H2 Build 22631.2428. Are there any log-files I can submit to help?

**Comment by riyasy** [comment]
1. Were you able to see the help window by clicking on the shortcut created on desktop?
2. If yes, Can you try launching the app from a folder which has only just one or two images? (Preferably simple JPEGs)
3. The log is present in "C:\Users\<userName>\AppData\Local\Temp\FlyPhotos\FlyPhotos.log". Need to delete this and rerun app to see if something is created. If app is not launching at all, I don't have hope something will be there in the log.
4. The installer with .Net runtime doesn't install anything system wide like in earlier days of .Net framework. With the new .Net features, it just has the dependent .Net DLLs inside the installation folder of Fly only. No system feature is touched.

**Comment by Skyyblaze** [comment]
1: I sadly don't have shortcuts on the Desktop but running the Fly .exe directly doesn't spawn anything either
2: I tried that now and there's no change in behavior
3: I see, yeah no log-file is created since I deleted the old one from Fly 1.0
4: Good to know!

Fly 1.0.0 works still without issues.

**Comment by riyasy** [comment]
Ok. I had released 1.0.2 yesterday. Please try that also when time permits.

Uninstall Fly from Add or remove programs in settings. Just make sure nothing exists in Program Files/RYFTools/Fly.
Install using latest 1.0.2. 

If normal operation is not working, 
As part of installation Help shortcut is created in desktop. Click that shortcut or directly execute the exe from program files. In either case Help window should be coming. If it is not coming, there is something going on which I need to check.

**Comment by Skyyblaze** [comment]
Sure thing and I actually tried 2.0 RC now and this works beautifully, thank you so much! Do you still want me to test 1.02 or is that obsolete at this point?

**Comment by riyasy** [comment]
1.x was done using WPF. 
2.x is done with latest technologies of Microsoft (WinRT, WinUI and Win2d) which is maturing currently. 
In WinRT, many APIs starts failing unexpectedly after calling them continuously. Also, many lower layer Image related APIs of the new SDK are slower than WPF. The caching speed of 2.x is around 50% slower than 1.x. 
2.x started based on your feature suggestion about Mica, Acrylic, but at last resulted in a full rewrite using latest technologies.

In short
- 1.x for caching performance, stability, and lower memory footprint or 
- 2.x for image pan zoom performance, animations and total aesthetics but suffer to be unstable until Microsoft matures the platform.

I am yet to decide which one to focus more. I would have pursued 2.x but for the issues in latest Windows SDK.

**Comment by riyasy** [comment]
I have dropped the Neo suffix for the 2.0 version and has updated the installer for 2.0.0 RC. Uninstall everything and install fresh. Please check and delete program files/RYFTools also.
- V1 will be called Fly Photos V1
- V2 version will be just called Fly Photos

**Comment by Skyyblaze** [comment]
Thanks, I did a clean install of 2.0 now and everything works great. I do understand your conundrum as a developer however in my opinion 2.0 seems to be more worthwhile, annoying about the SDK issues though, hopefully Microsoft gets this fixed. Maybe keep 1.0 around as a kind of FlyPhotos Lite?
